packages/react-components/src/components/timegraph-output-component.tsx

```diff
@@ -317,12 +317,11 @@ export class TimegraphOutputComponent extends AbstractTreeOutputComponent<Timegr
         return true;
     }
 
-    private doHandleSelectionChangedSignal(payload: { [key: string]: string }) {
+    private doHandleItemSelectedSignal(payload: { [key: string]: string; }): void {
         const startTimestamp = payload['startTimestamp'];
         const endTimestamp = payload['endTimestamp'];
         if (startTimestamp !== undefined && endTimestamp !== undefined) {
             const foundElement = this.findElement(payload);
-
             // Scroll vertically
             if (foundElement) {
                 // Expand parent
```

Comment:
When the existing "change selection range" function triggered, it searched for a state with matching timestamps.  If one was found, it ensured that it was visible within the view range.  

Since we don't want that happening every time the view range changes now, this logic was switched to the ItemPropertiesUpdatedSignal, both always happen at the same time when we click on an event in the Events Table (see change in selectAndNavigateToState in the previous comment).